estate/models/estate_property.py

```diff
@@ -0,0 +1,101 @@
+from dateutil.relativedelta import relativedelta
+from odoo import api, models, fields, exceptions
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    notes = fields.Html()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        default=fields.Date.today() + relativedelta(months=3), copy=False)
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(
+        readonly=True, copy=False, compute="_compute_selling_price_and_buyer")
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection([
+        ('north', 'North'),
+        ('south', 'South'),
+        ('east', 'East'),
+        ('west', 'West'),
+    ])
+    active = fields.Boolean(default=True)
+    state = fields.Selection([
+        ('new', 'New'),
+        ('offer_received', 'Offer Received'),
+        ('offer_accepted', 'Offer Accepted'),
+        ('sold', 'Sold'),
+        ('cancelled', 'Cancelled'),
+    ], default='new', required=True, copy=False)
+    property_type_id = fields.Many2one(
+        "estate.property.type", string="Property Type")
+    buyer_id = fields.Many2one("res.partner", string="Buyer",
+                               copy=False, compute="_compute_selling_price_and_buyer")
+    salesperson_id = fields.Many2one(
+        "res.users", string="Salesperson", default=lambda self: self.env.uid)
+    tag_ids = fields.Many2many("estate.property.tag", string="Tags")
+    offer_ids = fields.One2many(
+        "estate.property.offer", "property_id", string="Offers")
+    total_area = fields.Integer(
+        compute="_compute_total_area", string="Total Area (sqm)")
+    best_offer = fields.Float(
+        compute="_compute_best_offer", string="Best Offer")
```

Comment:
```suggestion
    best_offer = fields.Float(
        compute="_compute_best_offer")
```
Default string is OK